content/community/contents.lr

```diff
@@ -1,9 +1,9 @@
-_model: page
----
-_template: page-with-toc.html
----
-title: Join the CC Developer Community
----
+## \_model: page
```

Comment:
Please don't change the model, template, and title sections, this is Lektor's formatting.